tests/test_session_builder.c

```diff
@@ -298,9 +305,19 @@ START_TEST(test_basic_pre_key_v3)
 
     session_record *loaded_record = 0;
     session_state *loaded_record_state = 0;
+    session_state *state = 0;
     result = signal_protocol_session_load_session(alice_store, &loaded_record, &bob_address);
     ck_assert_int_eq(result, 0);
 
```

Comment:
I think maybe here you could just use `loaded_record_state`, whose value is set in line 321